service_extensions/callouts/add_header/README.md

```diff
@@ -0,0 +1,14 @@
+
+__Copyrights and Licences__
+
+Files using Copyright 2023 Google LLC & Apache License Version 2.0:
+* server.py contains gRPC server code
+* test_server.py contains client stub code to test the server.py
+
+Following files are auto-generated by grpc-tools based on protos/service.proto
+* service_pb2.pyi
```

Comment:
can we mention that they were generated by debian's grpc-tools and not the python pip one?